flair/models/text_classification_model.py

```diff
@@ -848,20 +845,29 @@ def _get_multi_label(self, label_scores) -> List[Label]:
 
     def _get_single_label(self, label_scores) -> List[Label]:
         conf, idx = torch.max(label_scores, 0)
-        label = self.label_dictionary.get_item_for_index(idx.item())
-
-        return [Label(label, conf.item())]
+        # TARS does not do a softmax, so confidence of the best predicted class might be very low.
+        # Therefore enforce a min confidence of 0.5 for a match.
+        if conf > 0.5:
```

Comment:
This could be problematic in some cases when TARS figures "True" for some sample with max score say 0.3. Let's leave it to the end-user to determine the threshold in a post-hoc manner.